packages/teams-js/src/private/otherAppStateChange.ts

```diff
@@ -0,0 +1,125 @@
+import { sendMessageToParent } from '../internal/communication';
+import { registerHandler, removeHandler } from '../internal/handlers';
+import { ensureInitialized } from '../internal/internalAPIs';
+import { ApiName, ApiVersionNumber, getApiVersionTag } from '../internal/telemetry';
+import { isNullOrUndefined } from '../internal/typeCheckUtilities';
+import { ErrorCode } from '../public';
+import { runtime } from '../public/runtime';
+
+/**
+ * @hidden
+ * @internal
+ * @beta
+ * Limited to Microsoft-internal use
+ *
+ * This capability contains the APIs for handling events that happen to other applications on the host
+ * *while* the developer's application is running. For example, if the developer wants to be notified
+ * when another application has been installed.
```

Comment:
just curious, will the event trigger also applies to other cases, like: app uninstall, or app updates/

Reply:
For now, no, but if we add those in the future they will be added here. I wanted to keep this limited to just what is needed for the current scenario.